### ISSUE TITLE
Elevated circulating amyloid A

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6220]
**Preferred term label:**
Elevated circulating amyloid A

**Synonyms**
Elevated serum amyloid A

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
FMF, see https://pubmed.ncbi.nlm.nih.gov/31941537/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
also aseptic meningitis

**Comment by pnrobinson** [comment]
added
